src/components/Karma/CircleGradient.tsx

```diff
@@ -53,7 +53,7 @@ export const CircleGradient = (props: {
           <Circle
             cx={props.size / 2}
             cy={props.size / 2}
-            r={r + 1}
+            r={r}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The radius `r` of the circle has been changed. This will affect the size of the circle. Please ensure this change is intended and does not distort the circle's appearance.
```diff
-            r={r + 1}
+            r={r}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->